app/algorithms/stv.ts

```diff
@@ -388,8 +380,8 @@ const calculateWinnerUsingSTV = (
 };
 
 // Round floats to fixed in output
-const handleFloatsInOutput = (obj: unknown) => {
-  const newObj = {};
+const handleFloatsInOutput = (obj: Record<string, number>) => {
+  const newObj: Record<string, number> = {};
```

Comment:
Thoughts on making this `typeof obj` as it should always mirror?

Reply:
I could, but there's no real difference since the type-checker makes sure the types match in any case